src/dll/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ target_include_directories(${PROJECT_NAME} PRIVATE ${DISKANN_MKL_INCLUDE_DIRECTO
 
 target_link_options(${PROJECT_NAME} PRIVATE /DLL /IMPLIB:${DISKANN_DLL_IMPLIB} /LTCG)
 target_link_libraries(${PROJECT_NAME} PRIVATE ${DISKANN_MKL_LINK_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} PRIVATE synchronization.lib)
```

Comment:
curious: was this missed related to the waitOnAddress change?

Reply:
Yes. when we build bann_diskann.dll in BANN repo, synchronization.lib is needed.